src/libguac/timestamp.c

```diff
@@ -29,13 +31,21 @@
 
 guac_timestamp guac_timestamp_current() {
 
+#undef CLOCK_MONOTONIC
 #ifdef HAVE_CLOCK_GETTIME
 
     struct timespec current;
 
-    /* Get current time */
+    /* Get current time, monotonically increasing. 
+     Using monotonic clock when available, to prevent backward running time */
+#ifdef CLOCK_MONOTONIC   
+    printf("MONOTONIC USED");
```

Comment:
Please remove the debugging statements. Looks like these `printf()`, the `#include <stdio.h>`, and the `#undef CLOCK_MONOTONIC` must have snuck in here during testing. ;)